README.md

```diff
@@ -145,7 +145,26 @@ export const echoCommand: Command<CtxWithEnv, Request, Toucan> = {
       },
     })
   },
-}
+};
+```
+You can also listen for webhook events from the Discord API (https://discord.com/developers/docs/events/webhook-events)
+Optional: Listen for an event that fires whenever someone authorizes your application.
+
+```ts
+import { ApplicationWebhookEventType } from 'discord-api-types/payloads';
+import type { Event } from 'workers-discord';
+
+import { CtxWithEnv } from '../env';
+
+export const appAuthEvent: Event<CtxWithEnv> = {
```

Comment:
```suggestion
const appAuthEvent: Event<CtxWithEnv> = {
```
I think this wants to be exported as default, based on how it's being imported later?

Reply:
As an aside, I think the exemplar command above also wants to be switched to being a default export?